guides/md/keras_cv/generate_images_with_stable_diffusion.md

```diff
@@ -30,6 +30,8 @@ that they offer.
 
 To get started, let's install a few dependencies and sort out some imports:
 
+pip install --upgrade keras-cv
```

Comment:
You can just add code markings here so that you don't have to regenerate the file (and a `!` in front)